src/qc.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+from scipy.stats import median_abs_deviation
+import anndata
+
+def human_mt_genes_ident(in_adata: anndata.AnnData) -> None:
```

Comment:
I know scanpy is a big fan of this, but when possible we try to avoid to modify a variable "in place" like you're doing here.

Better to create a copy, modify the copy and give back the modified version